plugins/nessus.rb

```diff
@@ -76,6 +76,7 @@ def commands
           "nessus_db_import" => "Import Nessus scan to the Metasploit connected database",
           "nessus_save" => "Save credentials of the logged in user to nessus.yml",
           "nessus_folder_list" => "List folders configured on the Nessus server",
+          "nessus_scanner_list" => "List the configured scanners on the Nessus server",
```

Comment:
I must have deleted the line on accident. Kind of weird since it's not actually close to where I put scan_workspace, but I didn't intend to remove it.